packages/react-native-gesture-handler/apple/RNGestureHandler.mm

```diff
@@ -389,9 +388,9 @@ - (void)sendEventsInState:(RNGestureHandlerState)state
 
 - (RNGHUIView *)findViewForEvents
 {
-  return
-      [self isKindOfClass:[RNNativeViewGestureHandler class]] && _actionType == RNGestureHandlerActionTypeNativeDetector
-      ? self.recognizer.view.superview
+  return _actionType == RNGestureHandlerActionTypeNativeDetector ||
+          _actionType == RNGestureHandlerActionTypeVirtualDetector
```

Comment:
Didn't we have a helper for this? Or was this Android?

Reply:
As far as I remember it was added only on android. I'll check that and add it if not present.